pkg/plugin/sdk/v1alpha2/plugin/interface.go

```diff
@@ -27,3 +33,40 @@ type Plugin interface {
 		promptRunner ioutils.PromptRunner,
 	) error
 }
+
+type CustomCommandFn (func(ctx context.Context, args []string) error)
+
+var fnMap map[string]CustomCommandFn = make(map[string]CustomCommandFn)
```

Comment:
Why do we need to save them globally?

Reply:
Definitely possible I didnt set this up properly here. Basically I needed this map to be shared between the calls of both `saveRunFunctions` and `executeRunFunction`

Reply:
This has been fixed now